src/kaleidoscope/key_events.h

```diff
@@ -72,7 +72,18 @@
  * currentState may be flagged INJECTED, which signals that the event was
  * injected, and is not a direct result of a keypress, coming from the scanner.
  */
-void handleKeyswitchEvent(Key mappedKey, HARDWARE_IMPLEMENTATION::KeyScannerDescription::KeyAddr key_addr, uint8_t keyState);
+void handleKeyswitchEvent(Key mappedKey, HARDWARE_IMPLEMENTATION::Description::KeyScannerDescription::KeyAddr key_addr, uint8_t keyState);
 DEPRECATED(ROW_COL_FUNC) inline void handleKeyswitchEvent(Key mappedKey, byte row, byte col, uint8_t keyState) {
   handleKeyswitchEvent(mappedKey, KeyAddr(row, col), keyState);
 }
+
+namespace kaleidoscope {
+namespace driver {
+
+template<>
+inline
+void BaseKeyScanner<HARDWARE_IMPLEMENTATION::Description::KeyScannerDescription>::handleKeyswitchEvent(Key mappedKey, HARDWARE_IMPLEMENTATION::Description::KeyScannerDescription::KeyAddr key_addr, uint8_t keyState) {
+  ::handleKeyswitchEvent(mappedKey, key_addr, keyState);
+}
+} // namespace driver
+} // namespace kaleidoscope
```

Comment:
We could also have this in Kaleidoscope.h but I found key_events.h to be more appropriate. Also I didn't want to pollute the already cramped Kaleidscope.h any further.